backend/fleet_management/middleware.py

```diff
@@ -0,0 +1,10 @@
+from django.utils import timezone
+from django.utils.deprecation import MiddlewareMixin
+
+from fleet_management.models import User
+
+
+class UpdateLastSeenMiddleware(MiddlewareMixin):
+    def process_request(self, request):
```

Comment:
shouldn't that be `__call__()`? Because that looks like a Pre 1.10 code according to docs:
https://docs.djangoproject.com/en/2.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware